lib/pages/general/SignUpInstructionsPage.dart

```diff
@@ -0,0 +1,154 @@
+    'assets/signup_instructions2.png',
+    'assets/signup_instructions3.png',
+    'assets/signup_instructions4.png',
+    'assets/signup_instructions5.png',
+    'assets/signup_instructions6.png',
+  ];
+
+  final List<String> descriptionsList = [
+    'Visit https://sis.auth.gr/old and login with your academic account.',
+    'Click on the top left menu button.',
+    'Click the second choice "Βαθμολογία" available on the menu.',
+    'Click again the second choice "Αναλυτική" available on the new menu that popped up.',
+    'Click the pdf icon "Εκτύπωση αναλυτικής βαθμολογίας σε pdf". A pdf file with your grades will be downloaded.',
+    'You are ready! You can now select the downloaded pdf file in the third stage of our app\'s signup process',
+  ];
+
+  @override
+  Widget body(GlobalKey<ScaffoldState> scfKey) {
+    return Container(
+      decoration: BoxDecoration(
+        gradient: LinearGradient(
+          begin: Alignment.topCenter,
+          end: Alignment.bottomCenter,
+          colors: [
+            Colors.blue,
+            Colors.white.withOpacity(0.20),
+          ],
+        ),
+      ),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          SizedBox(
+            height: 10.0,
+          ),
+          Center(
+            // Image and dots row stack widget
+            child: Stack(
+              children: <Widget>[
+                Container(
+                  decoration: BoxDecoration(
+                    image: DecorationImage(
+                      image: AssetImage(
+                        imgList[imgIndex],
+                      ),
+                      fit: BoxFit.contain,
+                    ),
+                  ),
+                  height: 300.0,
+                  width: 250.0,
+                ),
+                Positioned(
+                  top: 275.0,
+                  left: 15.0,
+                  right: 15.0,
+                  child: DotsRow(
+                    numberOfDots: imgList.length,
+                    imgIndex: imgIndex,
+                  ),
+                )
+              ],
+            ),
+          ),
+          SizedBox(
+            height: 15.0,
+          ),
+          InstructionsStepDescription(
+            imgIndex: imgIndex,
+            description: descriptionsList[imgIndex],
+          ),
+          SizedBox(
+            height: 15.0,
+          ),
+          // Buttons row
+          Padding(
+            padding: const EdgeInsets.only(left: 47.5, right: 47.5),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: <Widget>[
+                if (imgIndex != 0)
+                  _buildTextButton('Prev', _previousImage)
+                else
+                  SizedBox(),
+                if (imgIndex != imgList.length - 1)
+                  _buildTextButton('Next', _nextImage)
```

Comment:
Maybe we can remove the buttons and make it with swipes? And a final ok at the end.

Reply:
Commit with id 0a39764 now meets the proposed change.

https://user-images.githubusercontent.com/57541732/118834345-d3b9b600-b8ca-11eb-9435-518158e6dfa1.mp4